aleph-client/src/elections.rs

```diff
@@ -1,9 +1,10 @@
-use primitives::SessionIndex;
-pub use primitives::{CommitteeSeats, EraValidators};
+use primitives::{
```

Comment:
Please bump `version` in Cargo.lock for all modified crates

Reply:
Done.